nixd/include/nixd/Protocol/AttrSet.h

```diff
@@ -31,6 +32,13 @@ constexpr inline std::string_view AttrPathInfo = "attrset/attrpathInfo";
 constexpr inline std::string_view AttrPathComplete = "attrset/attrpathComplete";
 constexpr inline std::string_view OptionInfo = "attrset/optionInfo";
 constexpr inline std::string_view OptionComplete = "attrset/optionComplete";
+
+/// \brief ↩︎ Request information
+constexpr inline std::string_view BuiltinInfo = "attrset/builtinInfo";
```

Comment:
Also I wonder if we can just use `attrset/attrpathInfo` to complete such logic.

Reply:
I feel like I don't know enough about the protocol to say anything definitive, but primops and module system options seem quite different to me, so I wouldn't be surprised if you wanted to return different fields for them.